drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -284,6 +284,15 @@ def self.generate_enrollment_id
     generate_uuid
   end
 
+  def self.snapshot_enrollments(enrollments)
+    enrollments.map do |enrollment|
+      {
+        'enrollmentId': enrollment.id,
+        'relationshipToHoh': enrollment.relationship_to_hoh,
```

Comment:
we should snake case here as well. Suggested in other comment to move this to snapshot the Household if possible, since the household event table is snapshotting households